launcher/src/main.rs

```diff
@@ -621,6 +611,13 @@ fn shard_manager(
     // Copy current process env
     let mut envs: Vec<(OsString, OsString)> = env::vars_os().collect();
```

Comment:
This line was introduced in #621 , which means that the value of `envs` is overwritten and no longer contains the value of `PREDIBASE_API_TOKEN`.